plugins/utils_operators/test_file_operators.py

```diff
@@ -11,7 +11,7 @@
 from utils_operators.file_operators import DownloadFileOperator
 
 # init the url where the data will come from
-file_url = "https://contrib.wp.intra.prod-toronto.ca/app_content/tpp_measures/"
+file_url = "https://httpbin.org/html"
```

Comment:
This url will reliably never change - it seems like a silly choice, but I trust its response to stay constant more than anywhere else on the internet. Im open to suggestions here.